scene/main/viewport.cpp

```diff
@@ -363,9 +363,6 @@ void Viewport::_sub_window_grab_focus(Window *p_window) {
 		return;
 	}
 
-	int index = _sub_window_find(p_window);
-	ERR_FAIL_COND(index == -1);
-
```

Comment:
I would not remove this guard, which checks, that the argument `p_window` references to a non-existent window. The other newly introduced checks verify that nothing problematic happens during the event-callbacks.